CHANGELOG.md

```diff
@@ -295,7 +295,7 @@ Changelog for the Human Reference Atlas (HRA) User Interfaces
   - `footer`
   - `fullscreen`
   - `info-modal`
-  - `micro-tooltip`
+  - `plain-tooltip`
```

Comment:
Don't change this. We'll add new entries to the changelog once we update the versions